src/bytes.ts

```diff
@@ -1,19 +1,20 @@
-import { bytesToHex } from 'ethereum-cryptography/utils';
+import type { BytesLike, Hexable } from "@ethersproject/bytes";
 
-export type Bytes = Uint8Array;
+type Hex = BytesLike | Hexable | number | bigint;
+type HexString = string;
 
-export function compareBytes(a: Bytes, b: Bytes): number {
-  const n = Math.min(a.length, b.length);
+import {
+  isBytesLike,
+  arrayify as toBytes,
+  hexlify as toHex,
+  concat,
+  isBytes,
```

Comment:
I don't think this is never used. It feels like `isBytesLike` is enough.

Reply:
Right, I'll remove it